matador/plotting/magres_plotting.py

```diff
@@ -31,8 +31,9 @@ def plot_magres(
     signal_labels: Optional[Union[str, List[str]]] = None,
     signal_limits: Tuple[float] = None,
     line_kwargs: Optional[Union[Dict, List[Dict]]] = None,
+    flipxax: bool = False,
```

Comment:
I suggest we use the same name as matplotlib, and also make it `True` by default.

```suggestion
    invert_xaxis: bool = True,
```